scout/server/blueprints/variant/utils.py

```diff
@@ -89,8 +89,20 @@ def update_representative_gene(variant_obj: dict, variant_genes: List[dict]):
         variant_obj["first_rep_gene"] = None
 
 
+def update_transcript_mane(hgnc_transcript: dict, transcript: dict):
+    """Updates MANE key/values for a transcript in genome build 38."""
+    for key in ["mane_select", "mane_plus_clinical"]:
+        if hgnc_transcript.get(key):
+            transcript[f"{key}_transcript"] = hgnc_transcript[key]
+        else:
+            transcript.pop(f"{key}_transcript", None)
```

Comment:
The fix was finally pushed I see. Please test it again when you have time @alkc

Reply:
`dict.pop()` - huh, learned something new today!